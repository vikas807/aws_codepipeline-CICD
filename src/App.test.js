import { render, screen } from '@testing-library/react';
import App from './App';
jest.mock('maplibre-gl/dist/maplibre-gl', () => ({
  Map: () => ({}),
}));
test('renders learn react link', () => {
  const { container, rerender } = render(<App />);
 expect(container).toMatchSnapshot();
});