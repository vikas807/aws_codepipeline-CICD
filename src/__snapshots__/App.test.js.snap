// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div>
  <div
    class="App"
  >
    <h2>
       First test 
    </h2>
  </div>
</div>
`;
